lib/collection/insertMany.js

```diff
@@ -17,14 +17,14 @@ module.exports = function(Collection) {
 		options = options || {ordered: true};
 
 		if (!this._checkMethodSupport('insertMany')) {
-			return callback(MongoError.create({
+			return callback(new MongoError({
```

Comment:
We have a few options here:

 * Continue using deprecated `MongoError.create` method until the next major driver release.
 * Deprecate `mongo@2`.
 * Add a custom error class or factory function similar to `MongoError.create`:

```js
var err = new MongoError(options.message);
err.driver = options.driver;
```

Reply:
I don't think we should deprecate `mongo@2` because of a changed `new MongoError` signature.

Both other options seems fine.

I would suggest to continue using `MongoError.create` for now and worry about it later.
It's possible that we'll have to deprecate `mongo@2` anyway in order to support its next major release.

Reply:
Let's return `MongoError.create`, it seems the most reasonable way